scripts/wait-for-it.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
```

Comment:
add comment below here, where you got this from and presumably you converted the syntax from bash to /bin/sh

Reply:
done.